hw/drivers/trng/trng_da1469x/src/trng_da1469x.c

```diff
@@ -105,3 +106,23 @@ da1469x_trng_init(struct os_dev *dev, void *arg)
 
     return 0;
 }
+
+#ifdef MBEDTLS_ENTROPY_HARDWARE_ALT
+int
+mbedtls_hardware_poll(void *data, unsigned char *output, size_t len, size_t *olen)
```

Comment:
This would be way better if it was located in `hw/drivers/trng/src/mbed_entropy_alt.c` (or any other similar name) so that it would be automatically usable by all `trng` drivers.

Reply:
done.